bullet-train.zsh-theme

```diff
@@ -591,9 +594,17 @@ prompt_status() {
   fi
 
   local symbols
+  local ret_signal
   symbols=()
+  ret_signal=$(kill -l $(($RETVAL - 128)))
+
   [[ $RETVAL -ne 0 && $BULLETTRAIN_STATUS_EXIT_SHOW != true ]] && symbols+="✘"
-  [[ $RETVAL -ne 0 && $BULLETTRAIN_STATUS_EXIT_SHOW == true ]] && symbols+="✘ $RETVAL"
+  if [[ $RETVAL -gt 128 && $BULLETTRAIN_STATUS_EXIT_SIGNAL_SHOW == true ]]; then
+    # In Linux, return value grater than 128, i.e. 128+n, means fatal error of signal "n"
+    [[ $BULLETTRAIN_STATUS_EXIT_SHOW == true ]] && symbols+="✘ $ret_signal "
+  else
+    [[ $RETVAL -ne 0 && $BULLETTRAIN_STATUS_EXIT_SHOW == true ]] && symbols+="✘ $RETVAL"
+  fi
```

Comment:
Can we refactor those lines? This looks like some checks are redundant. It is a little bit of mess.
If we wrap it in one `if retval != 0` than I think it will be more readable.

Also in line 604, the space at the end is not necessary - this is the last string in 'symbols'